domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputHasDenominatorEqualToRuleClassifierProviderTest.kt

```diff
@@ -25,11 +26,17 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class FractionInputHasDenominatorEqualToRuleClassifierProviderTest {
-  private val WHOLE_NUMBER_123 = createWholeNumber(isNegative = false, value = 123)
-  private val FRACTION_1_OVER_2 = createFraction(isNegative = false, numerator = 1, denominator = 2)
-  private val FRACTION_2_OVER_4 = createFraction(isNegative = false, numerator = 2, denominator = 4)
-  private val NON_NEGATIVE_VALUE_1 = createNonNegativeInt(value = 1)
-  private val NON_NEGATIVE_VALUE_2 = createNonNegativeInt(value = 2)
+
+  private val WHOLE_NUMBER_123 =
+    createWholeNumber(isNegative = false, value = 123)
+  private val FRACTION_1_OVER_2 =
+    createFraction(isNegative = false, numerator = 1, denominator = 2)
+  private val FRACTION_2_OVER_4 =
```

Comment:
check the naming of variables in the reference file. 
example - `STRING_VALUE_TEST_UPPERCASE`

this could be `FRACTION_VALUE_TEST_1_OVER_2 `
`<type>_value_test_<what_to_test>`

Reply:
done